tests/test-cases/Group3-Docker-Compose/3-04-Docker-Compose-Fixtures.robot

```diff
@@ -0,0 +1,419 @@
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+V1 config
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/v1-config/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+V2 Dependencies
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/v2-dependencies/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+V2 full
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/v2-full/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+V2 simple
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/v2-simple/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+V3 full
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/v3-full/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Volume path interpolation
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/volume-path-interpolation/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Volume path
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/volume-path/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Volume
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/volume/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Volumes from container
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/volumes-from-container/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Volumes
```

Comment:
I don't know if this is the right place to comment since you didn't specify which yml file you use here.

For volumes, if you use `external` syntax for a volume, it has to be created before compose-up.

so something like 

```
docker volume create my_volume
echo volume_test_yml > docker-compose.yml
docker-compose up -d
```